packages/design-tokens/docs/pages/how-to-setup.stories.mdx

```diff
@@ -0,0 +1,65 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import { zenTheme, heartTheme } from '../../src/themes'
+import { Card } from '@kaizen/draft-card'
+import { Box } from "@kaizen/component-library";
+import { makeCSSVariableTheme } from '../../src/utils'
+import { ThemesCodeBlocks, SassVariablesCodeBlocks, CodeBlock, LinkToStory } from "../DocsComponents"
+import LinkTo from '@storybook/addon-links/react'
+import howToUseInSass from './how-to-use-with-sass.stories.mdx'
+import howToUseInReact from './how-to-use-in-react.stories.mdx'
+import howToSwitchThemes from './how-to-switch-themes.stories.mdx'
+
+<Meta title="How-to guides/Getting started" parameters={{ docsOnly: true }} />
```

Comment:
It would be good to make sure the filenames match the titles of each of our pages.